yasa/hypno.py

```diff
@@ -1425,7 +1489,15 @@ def hypno_consolidate_stages(hypno, n_stages_in, n_stages_out):
 #############################################################################
 
 
-def simulate_hypno(tib=90, sf=1 / 30, n_stages=5, trans_probas=None, init_probas=None, seed=None):
+def simulate_hypno(
+    tib=90,
+    n_stages=5,
```

Comment:
Nice catch!! No reason at all, I had just overlooked that. I removed them. Saves lot of redundant docstrings.

Reply:
Okay this was slightly trickier than I thought. Long story short, it's still a good idea and it's still done. But in the end I had too much code dedicated to handling unique-and-likely-rare cases where the `n_stages` is lower than user-specified transition probabilities. Rather than keeping that, I removed that option. You can still drop the `n_stages` if using default Metzner transition probabilities.

```python
simulate_hypno(n_stages=5)  # YES!
simulate_hypno(n_stages=2)  # YES!
simulate_hypno(n_stages=2, trans_probas=hyp5stages.transition_matrix()[1])  # NO!
simulate_hypno(n_stages=2, trans_probas=hyp5stages.consolidate_stages(2).transition_matrix()[1])  # YES!

simulate_hypno(n_stages=5).simulate_similar(n_stages=5)  # YES!
simulate_hypno(n_stages=5).simulate_similar(n_stages=4)  # NO!
simulate_hypno(n_stages=5).simulate_similar(n_stages=5).consolidate_hypno(4)  # YES!
```

I think this is probably cleaner anyways.

If you'd like, I could even remove the option to have <5 `n_stages` when using defaults, just for clarity/consistency? Instead of `simulate_hypno(n_stages=4)` users would have to do `simulate_hypno(n_stages=5).consolidate_stages(4)`. I kinda like the convenience of the former though.

Reply:
Same story for `freq`.

Reply:
I like the former option!

Reply:
Cool, that's what was implemented in my latest push. So this is ready for re-review then 👍